Scheduler/observing_blocks_maintel/AOS/BLOCK-T345.json

```diff
@@ -0,0 +1,79 @@
+{
+    "name": "AOSSurveyMode",
+    "program": "BLOCK-345",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "maintel/track_target.py",
+            "standard": true,
+            "parameters": {
+                "target_name": "$name",
+                "slew_icrs": {
+                    "ra": "$ra",
+                    "dec": "$dec"
+                },
+                "rot_value": "$rot",
```

Comment:
I think you might be better off using `rot_sky` here and `Sky` for `rot_type`.

Reply:
Yes .. did you update this already?

Reply:
I just updated this.